docs-kits/kits/PURIS Kit/page_adoption-view.md

```diff
@@ -1,232 +1,187 @@
+| -------------------------------------------------------------------------------------------------------------------------------------- | :-----: | :-----------: | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Item Stock](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.item_stock)                                 | v.2.0.0 |    CX-0122    | Item Stock includes materials in the outbound warehouse or inbound at the customer's site. Standardizing its definition and using digital twins for data sharing allows quick inventory updates across the supply chain, improving coordination and addressing supply shortages.                      |
+| [Short-Term Material Demand](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.short_term_material_demand) | v.2.0.0 |    CX-0120    | This exchange specifies the material a customer's production line needs from a supplier in the next four weeks. It adds context to regular orders, highlighting essential demands and extra requests for safety stocks. This detail helps suppliers plan and address potential shortages proactively. |
+| [Planned Production Output](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.planned_production_output)   | v.2.0.0 |    CX-0121    | This exchange shows the agreed but unmanufactured material quantity a supplier plans to produce for a customer. Standardizing this via digital platforms ensures clear communication of future production volumes, aligning supply chain expectations and improving responsiveness to market needs.   |
+| [Delivery Information](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.delivery_information)             |  v.200  |    CX-0118    | Delivery Information includes logistics details and metrics. It specifies shipment timing, location, quantity, and compares planned vs. actual departure and arrival times. This data, including tracking numbers and essential logistics info, helps coordinate and streamline deliveries.           |
+| [Days of Supply](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/io.catenax.days_of_supply)                         | v2.0.0  |    CX-0145    | The Days of Supply \(DoS\) standard measures how long inventory can sustain operations without replenishment. This metric helps manage inventory efficiently. The standard supports proactive decisions and ensures data accuracy across the supply chain.                                            |
```

Comment:
Here the aspect model should be v2.0.0 while the standard should be v1.0.0

Suggestion: update the static link to the currently standardized semantic model: https://github.com/eclipse-tractusx/sldt-semantic-models/tree/4d239fc5709f71f39c3cf13581b5bcf960905157/io.catenax.days_of_supply/2.0.0

Reply:
If the version refers to the standard, then it should be version 1.0.0

Reply:
Link has been adapted and column name clarified